### PR TITLE
Add ListFilesResponseSize in profile

### PR DESCRIPTION
**Pull request by OHerasym**
Add ListFilesResponseSize in profile
Add log debug level in if statement

Related:
[APPLINK-14369](https://adc.luxoft.com/jira/browse/APPLINK-14369)

### CODE REVIEW

**Comment by OHerasym** [comment]
Old pull request: [here](https://github.com/smartdevicelink/sdl_core/pull/319)

**Comment by OHerasym** [comment]
Please review: @malirod, @VVeremjova , @Kozoriz, @AMikhniev

File: src/components/application_manager/src/commands/mobile/list_files_request.cc [hunk 1]
```diff
@@ -76,15 +77,20 @@ void ListFilesRequest::Run() {
...
-      //In AppFile to application stored full path to file. In message required
-      //to write only name file.
-      //Plus one required for move to next letter after '/'.
-      (*message_)[strings::msg_params][strings::filenames][i++] =
-        it->first.substr(it->first.find_last_of('/') + 1);
+    std::string filename = it->first.substr(it->first.find_last_of('/') + 1);
+    // In AppFile to application stored full path to file. In message required
+    // to write only name file.
+    // Plus one required for move to next letter after '/'.
+    if (i < ::profile::Profile::instance()->list_files_response_size()) {
```
Comment (pestOO) [inline src/components/application_manager/src/commands/mobile/list_files_request.cc]: Cache this value please before cycle.
In future this reading could be from database and will increase performance.

**Comment by pestOO** [comment]
Update both ini files.
Discuss with require managers updating requirement documents according to this change.

**Comment by pestOO** [comment]
@OHerasym, 
Schema validation shall drop/block/split the RPC with wrong parameter value.
Why behaviors from the bug-ticket happens?
Investigate this because we possible only mask a problem.

**Comment by AGaliuzov** [comment]
@malirod, this PR has no any updates bu I see some coments

**Comment by OHerasym** [comment]
@AGaliuzov , this defect is blocked by question

**Comment by AGaliuzov** [comment]
@anosach-luxoft, have about merge this PR?

**Pull request merged**